### PR TITLE
[tests-only] Refactor regex for file/folder creation test step

### PR DESCRIPTION
**Pull request by SwikritiT**
Fixes: https://github.com/owncloud/owncloud-test-middleware/issues/69

This fixes the issue, for now, though it's not a really good solution. We need to come up with a better solution for this issue in future

### CODE REVIEW

File: src/stepDefinitions/filesContext.js [hunk 1]
```diff
@@ -100,7 +100,10 @@ Given('user {string} has renamed the following files', function(userId, table) {
 
 Given('user {string} has renamed file/folder {string} to {string}', webdav.move)
 
-Given('user {string} has created folder {string}', webdav.createFolder)
+Given(/user "([^"]*)" has created folder "(.+)"/gm, function(userId, folderName) {
+  folderName = folderName.replaceAll("\\", "")
```
Comment (kiranparajuli589) [inline src/stepDefinitions/filesContext.js:104]: a comment will be the best why we're doing this

**Comment by kiranparajuli589** [comment]
looks good :medal_sports:

**Review by kiranparajuli589 — COMMENTED** [review]
looks good :medal_military:

**Review by Talank — COMMENTED** [review]
LGTM

**Review by phil-davis — APPROVED** [review]

**Pull request merged**